app/serializers/item_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class ItemSerializer < ActiveModel::Serializer
+  attributes :id, :name
+
+  #belongs_to :list
```

Comment:
I don't think this use case justifies serializers, but feel free to keep using for the purposes of learning how to use them in this project... Also, if this were going to production, you would need to remove commented out code.